app/routes/my_pages.py

```diff
@@ -16,6 +17,10 @@ def get(self, hash_id=None):
 
         try:
             user = parse_hash(hashid=hash_id)
```

Comment:
The name of this variabel is a bit confusing, maybe rename it to simply personal_number?